runtime/near-vm-runner/src/runner.rs

```diff
@@ -108,3 +108,53 @@ impl VMKind {
         }
     }
 }
+
+/// Type returned by any VM instance after loading a contract and executing a
+/// function inside.
+#[derive(Debug, PartialEq)]
+pub enum VMResult {
+    /// Execution was not able to start because preconditions were not met.
+    NotRun(VMError),
+    /// Execution started but hit an error.
+    Aborted(VMOutcome, VMError),
+    /// Execution finished without error.
+    Ok(VMOutcome),
+}
+
+impl VMResult {
+    pub fn not_run(error: VMError) -> VMResult {
+        VMResult::NotRun(error)
+    }
```

Comment:
Is this method necessary? I think it is better to have just one way to construct the variant, even if it isn't consistent with helpers for some of the other variants.

Reply:
Removed